checklists/setting-up-meta-tags.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Setting up Meta Tags
+category: Front-End
+date: "2020-03-10"
+tags: ['meta', 'open-graph', 'twitter', 'handoff']
+description: The meta tags need to be defined on all projects.
+---
+
+- [ ] Add appropriate title to all pages  
+Some pages may not have their own title. In this case, the system should be responsible to give a default title. 
+
+- [ ] Add meta description  
+Meta description can be retrieved from the default description.
+
+- [ ] Make sure all relevant Open Graph tags have been provided  
+og:title, og:description, og:image, og:site_name, og:type, og:url must be provided. 
```

Comment:
Should we also cover the specification of the og image?

Reply:
I think this is the concern of Design team :) What do you think?

Reply:
Yes absolutely, but I think, you should give a sign for them. So that user can check whether it is correct or not.

Reply:
Sound reasonable. I'll add a warning to description. Also, let's put @akemalakcay to loop in case he may want to create a checklist which has this a to-do on it :)

Reply:
I'll take it to the consideration of the design team. Thanks! @alpcanaydin @ecempehlivan